src/internal/easee/model.go

```diff
@@ -0,0 +1,134 @@
+package easee
+
+import (
+	"time"
+)
+
+const (
+	ServiceName = "easee"
+)
+
+// BackPlate structure
+type BackPlate struct {
+	ID                string `json:"id"`
+	MasterBackPlateID string `json:"masterBackPlateId"`
+}
+
+// Charger structure
+type Charger struct {
+	ID            string    `json:"id"`
+	Name          string    `json:"name"`
+	Color         int       `json:"color"`
+	CreatedOn     string    `json:"createdOn"`
+	UpdatedOn     string    `json:"updatedOn"`
+	BackPlate     BackPlate `json:"backPlate"`
+	LevelOfAccess int       `json:"levelOfAccess"`
+	ProductCode   int       `json:"productCode"`
```

Comment:
Maybe product code is a model or serial number?

Reply:
Nope, it's just an arbitrary integer, doesn't look useful in any way.